src/components/Mission.js

```diff
@@ -34,6 +46,7 @@ Mission.propTypes = {
     id: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
     description: PropTypes.string.isRequired,
+    reserved: PropTypes.string.isRequired,
```

Comment:
Hi @ZeenatLawal,
your project is almost complete but there is a little change you need to make to move to the next milestone,

`reserved` propType can be a boolean instead of a string. Please check that you're receiving a boolean value from reserved property. Thank you.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.